index.html

```diff
@@ -450,9 +475,8 @@
         <a>PushManager</a> interface
       </h2>
       <p>
-        The <a>PushManager</a> interface defines the operations that enable <a title=
-        "webapp">webapps</a> to establish access to <a title="push service">push services</a>. Note
-        that just a single <a>push subscription</a> is allowed per <a>webapp</a>.
```

Comment:
I can't seem to find where you reformulate the note about having a single subscription per webapp in the diff. Did I miss it? If it's a deliberate change, we shouldn't include it in this PR.